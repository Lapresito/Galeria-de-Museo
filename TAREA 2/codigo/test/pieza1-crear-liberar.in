crearFecha 1/1/2001
crearPieza 7 pieza1 Carlos Nelson
liberarPieza
crearFecha 1/1/2001
crearPieza 7 pieza1 Luciana Diaz
liberarPieza
Fin
