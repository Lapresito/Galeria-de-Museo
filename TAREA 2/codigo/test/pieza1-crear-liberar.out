Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Fecha creada en forma exitosa.
2> Pieza creada en forma exitosa.
3> Pieza liberada en forma exitosa.
4> Fecha creada en forma exitosa.
5> Pieza creada en forma exitosa.
6> Pieza liberada en forma exitosa.
7> Fin.
